### ISSUE TITLE
Spoke Subnets are created as Public subnet 

### ISSUE THREAD

**Issue by nishanbrown** [oracle-quickstart/oci-arch-hub-spoke#11]
The spoke VCNs are created as Public subnet although the name indicates that Private subnet e.g. : spoke01_subnet_priv01 
![image](https://user-images.githubusercontent.com/47416629/123677031-9f62df00-d812-11eb-8b61-41a196207a22.png)

And according to reference architecture the spoke VCNs should not be accessible from the internet as well:
https://docs.oracle.com/en/solutions/hub-spoke-network/index.html#GUID-8295B649-5AF9-4CBD-A85E-153AB4916532

**Comment by oegentil** [comment]
Thanks for pointing that. I'll  fix that.

**Comment by nishanbrown** [comment]
Thank you so much @oegentil  for looking into this, this will help us a lot!

**Comment by olatunjiakingbade** [comment]
Thanks for looking into this @oegentil. This will be super helpful

**Comment by mohabh88** [comment]
Yes, I second @nishanbrown in his finding and would like to see this issue fixed soon @oegentil, please. It will help, of course
